compiler/x/i386/codegen/OMRRealRegister.hpp

```diff
@@ -147,6 +147,34 @@ class OMR_EXTENSIBLE RealRegister : public OMR::X86::RealRegister
          }
       }
 
+   static RegMask vectorMaskMask(RegNum idx)
+      {
+      switch (idx)
+         {
+         case OMR::RealRegister::NoReg:
+            return OMR::RealRegister::noRegMask;
+         case OMR::RealRegister::k0:
+            return OMR::RealRegister::k0Mask;
+         case OMR::RealRegister::k1:
+            return OMR::RealRegister::k1Mask;
+         case OMR::RealRegister::k2:
+            return OMR::RealRegister::k2Mask;
+         case OMR::RealRegister::k3:
+            return OMR::RealRegister::k3Mask;
+         case OMR::RealRegister::k4:
+            return OMR::RealRegister::k4Mask;
+         case OMR::RealRegister::k5:
+            return OMR::RealRegister::k5Mask;
+         case OMR::RealRegister::k6:
+            return OMR::RealRegister::k6Mask;
+         case OMR::RealRegister::k7:
+            return OMR::RealRegister::k7Mask;
+         default:
+            TR_ASSERT_FATAL(0, "vector mask mask valid for k1-k7 only");
```

Comment:
If true, should the `k0` case above assert then?

Reply:
The comment should say k0-k7